src/subdomains/core/liquidity-management/services/liquidity-management-rule.service.ts

```diff
@@ -81,6 +84,40 @@ export class LiquidityManagementRuleService {
     return LiquidityManagementRuleOutputDtoMapper.entityToDto(await this.ruleRepo.save(rule));
   }
 
+  async updateRuleSettings(
+    id: number,
+    dto: LiquidityManagementRuleSettingsDto,
+  ): Promise<LiquidityManagementRuleOutputDto> {
+    const rule = await this.ruleRepo.findOne({ id });
+
+    if (!rule) throw new NotFoundException(`Rule with id: ${id} not found.`);
+
+    const { reactivationTime } = dto;
+
+    rule.updateRuleSettings(reactivationTime);
+
+    return LiquidityManagementRuleOutputDtoMapper.entityToDto(await this.ruleRepo.save(rule));
+  }
+
+  //*** JOBS ***//
+
+  @Cron(CronExpression.EVERY_5_MINUTES)
+  async reactivateRules(): Promise<void> {
+    const rules = await this.ruleRepo.find({
+      status: LiquidityManagementRuleStatus.PAUSED,
+      reactivationTime: Not(IsNull()),
+    });
+
+    for (const rule of rules) {
+      if (rule.shouldReactivate()) {
+        rule.reactivate();
+        await this.ruleRepo.save(rule);
+
+        console.log(`Reactivated liquidity management rule ${rule.id}`);
```

Comment:
It would be cool to have an email notification (equivalent to 'rule paused' mail).

Reply:
done